src/sentry/projects/services/project/impl.py

```diff
@@ -132,6 +132,11 @@ def create_project_for_organization(
                 sender=self.create_project_for_organization,
                 user_id=user_id,
             )
+            # Add electron symbol server by default to both electron and javascript-electron projects
+            if project.platform and project.platform in ["electron", "javascript-electron"]:
+                project.update_option(
+                    "sentry:builtin_symbol_sources", ["ios", "microsoft", "electron"]
+                )
```

Comment:
It looks like the `project_created` signal is intentionally sent at the very end just before returning the project. Could you reorder this accordingly?